src/components/donation/Steps/DonateMethods/Stocks/Form.tsx

```diff
@@ -1,10 +1,11 @@
 import { yupResolver } from "@hookform/resolvers/yup";
-import { Field } from "components/form";
-import { FormProvider, UseFormReturn, useForm } from "react-hook-form";
+import { Field, Form as FormContainer } from "components/form";
```

Comment:
@ap-justin what do you think about renaming the `Form` component itself to `FormContainer`? It would make it clearer what the component's purpose it

Reply:
i prefer to be it just `Form` as its api is just extended `<form/>` , `FormContainer`( just renamed (not a good name) here to not conflict with the component name `function Form`)  sounds like a wrapper to an actual form 

i.e.
```jsx
<FormContainer>
  <form> 
```